rook/toolbox-deploy.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: rook-ceph-tools
+  namespace: rook-ceph # namespace:cluster
```

Comment:
```suggestion
  namespace: rook-ceph
```

Reply:
I simply copied the file from here: https://github.com/rook/rook/blob/master/deploy/examples/toolbox.yaml#L5

Unsure why they added the comment there, but since I didn't change anything else, it felt weird to change only this.